examples/functions/trip/TR_ParallelService_OeBB_wrong.xml

```diff
@@ -0,0 +1,243 @@
+										<!-- the PublishedService changes in the middle, therefore having this PublishedServiceName and the train number
+										for the whole Leg would be misleading. See correct example -->
+									</PublishedServiceName>
+									<TrainNumber>123123</TrainNumber>
+									<siri:VehicleRef>A</siri:VehicleRef>
+									<OperatorRefs>
+										<OperatorRef>at:oebb</OperatorRef>
+									</OperatorRefs>
+									<DestinationText>
+										<Text>Hamburg Altona</Text>
+									</DestinationText>
+								</Service>
+								<ParallelService>
+									<FromStopSequenceNumber>1</FromStopSequenceNumber>
+									<ToStopSequenceNumber>5</ToStopSequenceNumber>
+									<JourneyRef>NJ-OeBB-1230123-123111</JourneyRef>
+									<siri:LineRef>OeBB:NJ12311</siri:LineRef>
+									<siri:DirectionRef>H</siri:DirectionRef>
+									<Mode>
+										<PtMode>rail</PtMode>
+										<siri:RailSubmode>nightRail</siri:RailSubmode>
+										<Name>
+											<Text xml:lang="de">Zug</Text>
+										</Name>
+										<ShortName>
+											<Text xml:lang="de">NJ</Text>
+										</ShortName>
+									</Mode>
+									<PublishedServiceName>
+										<Text xml:lang="de">NJ 490</Text>
+									</PublishedServiceName>
+									<TrainNumber>123123</TrainNumber>
+									<siri:VehicleRef>A</siri:VehicleRef>
+								</ParallelService>
+								<ParallelService>
+									<!-- the other train component going to Amsterdam, AT part -->
+									<FromStopSequenceNumber>1</FromStopSequenceNumber>
+									<ToStopSequenceNumber>5</ToStopSequenceNumber>
+									<JourneyRef>NJ-OeBB-120123-123112</JourneyRef>
+									<siri:LineRef>OeBB:NJ12312</siri:LineRef>
+									<siri:DirectionRef>H</siri:DirectionRef>
+									<Mode>
+										<PtMode>rail</PtMode>
+									</Mode>
+									<PublishedServiceName>
+										<Text>NJ 40490</Text>
+									</PublishedServiceName>
+									<TrainNumber>1231012</TrainNumber>
+									<siri:VehicleRef>B</siri:VehicleRef>
+									<DestinationText>
+										<Text>Amsterdam</Text>
+									</DestinationText>
+								</ParallelService>
+								<ParallelService>
+									<!-- the German part of the train to Hamburg -->
+									<FromStopSequenceNumber>5</FromStopSequenceNumber>
+									<ToStopSequenceNumber>12</ToStopSequenceNumber>
+									<JourneyRef>NJ-OeBB-120123-1231122</JourneyRef>
+									<siri:LineRef>OeBB:NJ12312</siri:LineRef>
+									<siri:DirectionRef>H</siri:DirectionRef>
+									<Mode>
+										<PtMode>rail</PtMode>
+									</Mode>
+									<PublishedServiceName>
+										<Text>NJ 590</Text>
+									</PublishedServiceName>
+									<TrainNumber>123123</TrainNumber>
+									<siri:VehicleRef>A</siri:VehicleRef>
+									<DestinationText>
+										<Text>Hamburg Altona</Text>
+									</DestinationText>
+								</ParallelService>
+								<ParallelService>
+									<!-- the German part of the train to Amsterdam, only to Nürberg afterwards separated -->
```

Comment:
What does "only to Nürberg afterwards separated" mean? Do you mean "only to Nürnberg after being separated"?

Reply:
changed.